.github/workflows/codeowners-validation.yaml

```diff
@@ -2,19 +2,38 @@ name: "CodeOwners Validation"
 
 on:
   pull_request:
-    paths:
-      - ".github/CODEOWNERS"
-      - "CODEOWNERS"
+    # Path filtering doesn't work on org-wide rulesets
+    # paths:
+    #   - ".github/CODEOWNERS"
+    #   - "CODEOWNERS"
```

Comment:
Should we just remove then?

Reply:
Kinda wanted it there so people understand why the filtering job is necessary. I can make the comment better though.

Reply:
Ah ok, I think this works as-is then.

Reply:
Better comment: dec16e36c5bf79db09abb3399856949e0cb20d7f